src/Microsoft.Azure.WebJobs.Host/Hosting/WebJobsHostExtensions.cs

```diff
@@ -109,5 +111,46 @@ public static IHostBuilder UseExternalStartup(this IHostBuilder builder, IWebJob
 
             return builder;
         }
+
+        public static IHostBuilder ConfigureWebJobsFastLogging(this IHostBuilder builder, IEventCollectorFactory fastLogger)
```

Comment:
I agree! I would like to remove this. For this change, I'm just moving the method to the right extension type.

Reply:
Application Insights is a wonderful tool. Unfortunately, it is not free. Table logging provides a cost effective alternative to AI even though it may not be a feature rich. Those that are trying to minimize  their budget (non-profits) are grateful to those who take cost into consideration when making fundamental architectural decisions in the framework.

Reply:
@jucchytil with the new model, the removal of this just means that we're removing custom logging code maintained in the WebJobs codebase. You'd still have the ability to plug in different logging providers and this would not lock you into using Application Insights.

Reply:
Thank you so much fabio for taking the time to respond. It simply amazes me that I can 1) become aware of new feature development efforts such as this and 2) participate in its discussion. Please continue to engage newbies such as myself. To me, I find it humbling to be in the presence of those working on a product I care about. I look forward to learning how to plug in the different logging providers you are speaking of.